README.md

```diff
@@ -2,6 +2,10 @@
 
 <h5 align="center">A command-line Git information tool written in Rust</h5>
 
+<p align="center">
+	English | <a href="./README.ja.md">日本語</a>
+</p>
```

Comment:
@o2sh By the way, my review request was mainly because I was curious about your opinion on this piece, and if it should be adjusted or included in the Japanese README. I should have mentioned that earlier 😅

Reply:
No worries :smile:, 
This is pretty much what I had in mind. I would probably drop the `English` and move this section below GitHub badges.

```suggestion
<p align="center">
	[<a href="./README.ja.md">日本語</a>]
</p>
```